kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/storage/ProducerStateManager.java

```diff
@@ -178,7 +178,7 @@ void updateAbortedTxnsPurgeOffset(long abortedTxnsPurgeOffset) {
     }
 
     long maybePurgeAbortedTx() {
-        if (mapEndOffset == -1) {
+        if (mapEndOffset == -1 || kafkaTxnPurgeAbortedTxnIntervalSeconds <= 0) {
             return 0;
         }
         long now = System.currentTimeMillis();
```

Comment:
I find the `deltaFromLast` has been divided by 1000 twice. We should only divide it once.

Reply:
It seems that this issue had been fixed.